src/layouts/index.tsx

```diff
@@ -119,41 +106,69 @@ const Layout = ({ children }) => (
         <Footer>
         <Location>
           {({ location }) => {
-            const index = LINKSPATH.findIndex(path => path.path === location.pathname)
-            if(location.pathname === "/print/"){
-              return (
-                <ButtonContainer>
-                 <ButtonLinkGreen to="/screen-2/">Return to Results</ButtonLinkGreen>
-                 <ButtonLink to="/screen-2a/">Continue to Benefit Formula</ButtonLink>
-                </ButtonContainer>
-              )
-            }
-            if(index === -1){
-              return null;
-            }
-            if(index === LINKSPATH.length -1){
-              return (
-              <ButtonContainer>
-              <ButtonLinkGreen to={LINKSPATH[index -1].path}>
-               {`Previous: ${LINKSPATH[index -1].label[0] + LINKSPATH[index -1].label.slice(1).toLowerCase()}`}
-              </ButtonLinkGreen>
-              <ButtonLink to="/">Go Home</ButtonLink>
-              </ButtonContainer>
-              )
-            }
-            if(index === 0 ){
-              return (
-              <ButtonContainer>
-              <ButtonLink to="/prescreen-1a/">Get Started</ButtonLink>
-              </ButtonContainer>
-              )
+            const index = LINKSPATH.findIndex(path => (
+              path.path === location.pathname
+            ));
+            let labelLeft, labelRight, urlLeft, urlRight;
+
+            if (index === -1) return;
+            if (index === 0) {
+              labelLeft = ""
+              labelRight = "Get Started";
+              urlLeft = ""
+              urlRight = "/prescreen-1a/";
+            } else if (location.pathname === "/print/") {
+              labelLeft = "Return to Results";
+              labelRight = "Continue to Benefit Formula";
+              urlLeft = "/screen-2/";
+              urlRight = "/screen-2a/";
+            } else if (index === LINKSPATH.length - 1) {
+              labelLeft = `Previous: ${
+                LINKSPATH[index - 1].label[0] +
+                LINKSPATH[index - 1].label.slice(1).toLowerCase()
+              }`;
+              labelRight = "Go Home";
+              urlLeft = LINKSPATH[index - 1].path;
+              urlRight = "/";
+            } else {
+              labelLeft = `Previous: ${
+                LINKSPATH[index - 1].label[0] +
+                LINKSPATH[index - 1].label.slice(1).toLowerCase()
+              }`;
+              labelRight = `Next: ${
+                LINKSPATH[index + 1].label[0] +
+                LINKSPATH[index + 1].label.slice(1).toLowerCase()
+              }`;
+              urlLeft = LINKSPATH[index - 1].path;
+              urlRight = LINKSPATH[index + 1].path;
             }
+
+            let labelLeftMobile = index === 0 ? "" : "PREV";
+            let labelRightMobile = (
+              index === 0 ? "START" :
+              index === LINKSPATH.length - 1 ? "HOME" :
```

Comment:
```
const indexOfFirstPage = index === 0;
const indexOfLastPage = index === LINKSPATH.length - 1
```
and then use these in ternaries

Reply:
Refactored accordingly (used `isPageFirst`/`isPageLast` to indicate boolean nature).